components/ipfs/ipfs_utils.cc

```diff
@@ -63,11 +52,68 @@ base::span<const uint8_t> DecodeVarInt(base::span<const uint8_t> from,
   return from.subspan(it - from.begin());
 }
 
-GURL AppendLocalPort(const std::string& port) {
-  GURL gateway = GURL(ipfs::kDefaultIPFSLocalGateway);
-  GURL::Replacements replacements;
-  replacements.SetPortStr(port);
-  return gateway.ReplaceComponents(replacements);
+bool ParseCIDAndPathFromIPFSUrl(const GURL& url,
+                                std::string* cid,
+                                std::string* path) {
+  if (!url.SchemeIs(kIPFSScheme) && !url.SchemeIs(kIPNSScheme)) {
```

Comment:
I don't see calls to it in ipfs_utils_unittest. Either in test or in comment above there should be some examples of whati is being parsed and what is the expected result